core/src/test/java/io/questdb/cairo/TableReadFailTest.java

```diff
@@ -144,8 +144,8 @@ public void testReloadTimeout() throws Exception {
                 mem.close();
                 mem.close();
 
-                // make sure reload functions correctly
-                Assert.assertFalse(reader.reload());
```

Comment:
i think this test escalated, could we fix it by returning txn to original value, rather than artificially to `txn+2` ?

Reply:
nah, ignore it, too convoluted